### DISCUSSION TITLE
Minor CSS bug with codeblock highlighting

### DISCUSSION THREAD

**Discussion by flapili** [fastapi/fastapi#7224] [Questions]
Hi, there is a style bug on the online documentation.
The highlighting don't fill all the line when the codeblock have a scrollbar

![Screenshot_2020-08-11-17-50-14-572_com opera browser](https://user-images.githubusercontent.com/29481952/89919542-b1885e80-dbfb-11ea-8b3c-79679b3e31c7.jpg)

Unfortunaly I can't try to fix the CSS I don't have computer for a few more weeks

**Comment by ycd** [comment]
This is a mobile related issue with markdown_extensions, doesn't belongs to FastAPI

We use `hl_lines` when adding a highlight to a line

Example:

```Python
> ``` Python hl_lines="4  9 10 11 12  25 26 27"
> some_code_here
> ```
```

It adds highlighting to the whole line so the problem probably belongs to `markdown_extensions` in [mkdocs.yml](https://github.com/tiangolo/fastapi/blob/master/docs/en/mkdocs.yml)

**Comment by flapili** [comment]
Oops my Bad,
I'll open an issue on the mkdocs repo

**Comment by tiangolo** [comment]
Thanks for the help here @ycd ! :clap: :bow: 

Thanks for reporting back and closing the issue @flapili  :+1:
